contributors/ZoranIT.md

```diff
@@ -0,0 +1,13 @@
+2019-05-22
```

Comment:
This should be in the xibo main repository (see https://github.com/xibosignage/xibo/tree/master/contributors)